exercises/alice_russell/ex02_oo_basics/src/Circle.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+#include "Shape.h"
+
+
+
+class Circle : public Shape
+{
+private:
+	const double radius;
+public:
+	Circle(double r) :radius(r) {}
+	double find_perimeter() const;
```

Comment:
In modern C++ you will see compilers warn that methods like this can be marked "override". This essentially means you would write `double find_perimeter() const override;` to indicate that you are intending to override the method specified in `Shape`. That way the compiler can spot occasions where parameters might be incorrect and a function does not override as intended.